packages/valory/skills/hello_world_abci/behaviours.py

```diff
@@ -193,6 +207,8 @@ def async_act(self) -> Generator:
         else:
             message = ":|"
 
+        message += f" - The owner's address is {self.shared_state.owner_address}"
```

Comment:
No need to read from the shared state. Just read from the Params.

Reply:
Cool, then maybe I misunderstood what the exercise meant there. 

![Screenshot by Dropbox Capture](https://github.com/kongzii/hello-world/assets/15619339/f89c74d2-65bd-4dc6-901a-aac0e1112407)